sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeed.cs

```diff
@@ -234,8 +188,15 @@ internal DataFeedDetail GetDataFeedDetail()
                 detail.FillMissingPointValue = MissingDataPointFillSettings.CustomFillValue;
             }
 
-            Administrators = detail.Admins;
-            Viewers = detail.Viewers;
+            foreach (var admin in Administrators)
+            {
+                detail.Admins.Add(admin);
+            }
+
+            foreach (var viewer in Viewers)
+            {
+                detail.Viewers.Add(viewer);
+            }
```

Comment:
Found this by chance. Fixes https://github.com/Azure/azure-sdk-for-net/issues/17719. We were doing:
```cs
dataFeed.Property = codeGenModel.Property
```
but we should be doing the opposite.

Reply:
You could also do `detail.Admins.AddRange(Administrators)`

Reply:
Unfortunately, `Admins` and `Viewers` are `IList` references, so they don't have the `AddRange` method (available only in `List`).